### ISSUE TITLE
all_profile_res.empty() assert faild. can't find suitable algorithm for 0

### ISSUE THREAD

**Issue by 27ccc** [Pointcept/PointTransformerV3#176]
用到PointTransformerV3的时候遇到RuntimeError: /io/build/temp.linux-x86_64-cpython-310/spconv/build/core_cc/src/csrc/sparse/convops/convops/ConvTunerSimple/ConvTunerSimple_tune_and_cache.cc(103)
!all_profile_res.empty() assert faild. can't find suitable algorithm for 0应该怎么解决呢
我的环境是pytorch 2.8.0 cu12.8

**Comment by Kevinpyx** [comment]
Same here. It ran well during training but this error occurred during eval. I was able to bypass the error by setting all the spconv module to training mode during evaluation. It worked, but it's not proper fix of the issue.

**Comment by Gofinge** [comment]
Hey, try to rebuild the spconv library or install the right pre-built version of spconv

**Comment by kaguramena** [comment]
you can check this [https://github.com/traveller59/spconv/issues/563](url) i delete the model.eval() when i use bf16 and it fixed
